src/renderer/components/generic/navbar/useNavbar.ts

```
@@ -0,0 +1,400 @@
+          } catch (fetchError) {
+            // If fetching fails for a user, return null to filter it out
+            return null;
+          }
+        });
+
+        const playerDataResults = await Promise.all(playerDataPromises);
+
+        // Filter out null results and create the map
+        const newPlayerInfoMap: { [key: string]: PlayerData } = {};
+        playerDataResults.forEach((result) => {
+          if (result && result.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)